CliMAgen/examples/context2dturbulence/Experiment_64x64.toml

```diff
@@ -6,7 +6,7 @@ rngseed           = 123
 nogpu             = false
 
 [data]
-batchsize         = 64
+batchsize         = 4
```

Comment:
since we resize the 64x64 images to 512, we need to reduce the batchsize.

Reply:
yep! :)